plugins/flatpak/gs-flatpak-transaction.c

```diff
@@ -62,6 +62,41 @@ gs_flatpak_transaction_set_no_deploy (FlatpakTransaction *transaction, gboolean
 	g_object_notify (G_OBJECT (self), "no-deploy");
 }
 
+/* Checks if a ref is a related ref to one of the installed ref.
+ * If yes, return the GsApp corresponding to the installed ref,
+ * NULL otherwise.
+ */
+static GsApp *
+check_if_related_ref_is_installed (FlatpakTransaction *transaction,
+				    FlatpakTransactionOperation *operation)
```

Comment:
I think the method name is confusing given it returns the main app ref for a related ref. I would probably rename it to something like `get_main_app_for_related_ref`.

Also nit: missing space before `FlatpakTransactionOperation`.